libqaul/service/files/src/lib.rs

```diff
@@ -1,7 +1,32 @@
-#[cfg(test)]
-mod tests {
-    #[test]
-    fn it_works() {
-        assert_eq!(2 + 2, 4);
+//! `qaul.net` filesharing service
+
+use qaul::{Qaul, QaulResult, UserAuth};
+use identity::Identity;
+
+/// A typed file that can be sent across the network
+pub struct File {
+    mime: String,
```

Comment:
might make sense to use the hyperium `mime` crate?
https://github.com/hyperium/mime